pkg/image/utilities/utilities.go

```diff
@@ -541,15 +558,20 @@ if [[ $help ]]; then
     exit 0
 fi
 
+if [ ! -z "${source_registry}" ]; then
+    source_registry="${source_registry}/"
+fi
```

Comment:
same comment as above; we should either fail out with an invalid source registry or automatically trim the trailing slash if we see it in the input.

also, `target_registry` does not assume a trailing slash so neither should `source_registry`